style.css

```diff
@@ -1503,3 +1507,435 @@
   font-size: 23px;
   margin-left: 18px;
 }
+
+@media only screen and (min-width: 768px) {
+  .toolbar {
+    width: 100vw;
+  }
+
+  .menu-bar {
+    display: flex;
+    width: fit-content;
+    right: 120px;
+  }
+
+  .menu-bar img {
+    display: none;
+  }
+
+  .nav-menu {
+    display: flex;
+    list-style-type: none;
+  }
+
+  .nav-menu li {
+    color: #fff;
+    margin-left: 80px;
+  }
+
+  .nav-menu li a {
+    color: #fff;
+    text-decoration: none;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .headline {
+    width: 100vw;
+    height: 609px;
+    flex-direction: row;
+    background-image: url(./imgs/GroupBig.svg);
+  }
+
+  .headline-text {
+    order: 1;
+    align-items: center;
+    padding: 102px 0;
+    gap: 24px;
+    position: absolute;
+    width: 566px;
+    height: 739px;
+    left: 143px;
+    top: 72px;
+  }
+
+  .headline .socials ul {
+    flex-direction: column;
+    gap: 24px;
+  }
+
+  .headline .socials li i {
+    font-size: 24px;
+  }
+
+  .headline .primary {
+    width: 566px;
+    height: 160px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    color: #f55800;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .headline .supporting {
+    width: 566px;
+    height: 24px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .headline .description {
+    width: 566px;
+    height: 123px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+    flex: none;
+    order: 2;
+    flex-grow: 0;
+  }
+
+  .recent-works {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 130px 24px;
+    gap: 114px;
+    position: absolute;
+    width: 100vw;
+    height: 1805px;
+    top: 875px;
+  }
```

Comment:
You have done great at implementing the project cards. A closer look at the figma design shows that there're two form that the project cards take. One is the normal display of the card without the "See Project" button, the other is the way each card is expected to look when hovered where the "See Project" button appears. Kinslt implement this too inorder to c=foolow the design.

Card normal display:
![image](https://user-images.githubusercontent.com/20554583/215264092-28ca58a9-bfcb-4b34-a1e2-22e8226c3957.png)

Card when hovered:
![image](https://user-images.githubusercontent.com/20554583/215264101-15762033-2753-4c0b-89de-cf328459027f.png)

Reply:
Done.